core/comment.js

```diff
@@ -271,7 +302,22 @@ Blockly.Comment.prototype.disposeBubble_ = function() {
     Blockly.Warning.prototype.disposeBubble.call(this);
     return;
   }
-
+  if (this.onMouseUpWrapper_) {
+    Blockly.unbindEvent_(this.onMouseUpWrapper_);
+    this.onMouseUpWrapper_ = null;
```

Comment:
Do we need to set these to null?

Reply:
Since we're disposing of the whole comment, I think it's ok to remove that line. If it were inside a something like a field editor dispose it's probably safer to set it to null, but that's probably matter of opinion.

Fixin'!

Reply:
Yep!

Reply:
My bad, completely read this wrong. I think it is probably safer to leave this here because the comment is still alive, we're just disposing of the bubble. This way it works the same as when fields dispose of their editor listeners.

Reply:
Okay that sounds good to me. Note to self, we should do the same in all the fields (eg: [here](https://github.com/google/blockly/blob/develop/core/field_textinput.js#L363)) since as you said we're not disposing of the entire object when widgetDispose_ (or the like) is called .

Reply:
Already on it =D